windows/src/desktop/kmshell/xml/splash.xsl

```diff
@@ -49,7 +49,7 @@
   </div>
 
   <div id="version"><xsl:value-of select="/Keyman/Version"/>
-    <span id="beta"><xsl:value-of select="$locale/String[@Id='S_BETA']"/></span>
+    <span id="beta"><xsl:value-of select="$locale/string[@name='S_BETA']"/></span>

```

Comment:
1. I was expecting to find `S_BETA` in strings.xml but I didn't find it.
2. Will there also be an entry for `S_ALPHA`?

Reply:
This `S_BETA` reference is deleted in the next PR (#3428)